mods/stairs/init.lua

```diff
@@ -32,7 +32,11 @@ local function rotate_and_place(itemstack, placer, pointed_thing)
 	if placer then
 		local placer_pos = placer:get_pos()
 		if placer_pos then
-			param2 = minetest.dir_to_facedir(vector.subtract(p1, placer_pos))
+			local vec = vector.subtract(p1, placer_pos)
```

Comment:
`placer_pos` is the feet position of the player; you should probably use the placer's look dir instead (`player:get_look_dir()`).